### PR TITLE
Use xkb layout/variant from host X

### PR DESCRIPTION
**Pull request by peppsac**
@autra r?

[Review on Reviewable](https://reviewable.io/reviews/phoxygen/b2gian/4)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -132,9 +134,11 @@ ${B2G_PROFILE_PATH}:
 # gaia profile there
 run: ${BUILD_DIR}/.runtime_deps_ready ${BUILD_DIR}/gecko/dist/b2g ${B2G_PROFILE_PATH} ${BUILD_DIR}/gaia/profile
 	cp -aT ${BUILD_DIR}/gaia/profile ${B2G_PROFILE_PATH}
-	cd ${BUILD_DIR}/gecko/dist/b2g/ && \
-	startx ./b2g -no-remote -profile ${B2G_PROFILE_PATH} --screen ${RESOLUTION} -- /usr/bin/Xephyr \
-		-title "RoninOS" -ac -br -noreset -screen ${RESOLUTION}
+	(/usr/bin/Xephyr -title "Ronin OS" -ac -br -noreset -screen ${RESOLUTION} -displayfd 5 5>&1 2>/dev/null) | while read D; do \
```
Comment (autra) [inline Makefile]:
``` bash
startx ./b2g -no-remote -profile ${B2G_PROFILE_PATH} --screen ${RESOLUTION} -- /usr/bin/Xephyr \
        -title "RoninOS" -ac -br -noreset -screen ${RESOLUTION} \
                -keybd ephyr,xkbmodel=evdev,xkblayout=${XKB_LAYOUT},xkbvariant=${XKB_VARIANT}
```

Do I win with this one, this time? ;-)

Comment (peppsac) [inline Makefile]: Well they're not strictly equivalent. Using my version:

```
rules:      evdev
model:      pc105
layout:     fr,us,fr
variant:    latin9,,oss
```

With yours:

```
rules:      evdev
model:      pc105
layout:     fr
variant:    latin9
```

Comment (peppsac) [inline Makefile]: But I'll change for your solution anyway (easier to understand)

**Comment by peppsac** [comment]
r?

**Comment by autra** [comment]
r+ ! :-)

**Pull request merged**